src/supermarket/config/routes.rb

```diff
@@ -30,7 +30,7 @@
       post "/cookbook-versions/publish_evaluation" => "quality_metrics#publish_evaluation", as: :cookbook_versions_publish_evaluation, constraints: proc { Feature.active?(:fieri) }
       post "/cookbook-versions/license_evaluation" => "quality_metrics#license_evaluation", as: :cookbook_versions_license_evaluation, constraints: proc { Feature.active?(:fieri) }
```

Comment:
Is this redundant in favour of foodcritic_evaluation?

Reply:
Yes, We are removing all license evaluation references too since cookstyle cop - https://docs.chef.io/workstation/cookstyle/chef_sharing_invalidlicensestring/ is covering the same